unitTestSupportFunctions/calculateLift.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate lift force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fy : Force (lift) on the body in y-direction
+%
+%% Function main body
+function Fy = calculateLift (FComplete, fldMsh, homDBC, parameters)
+
+    % Get node indices on the body
+    indices = getBodyNodes(fldMsh, homDBC);
+    
+    % Reshape the input forces to [Fx, Fy, Fz]
+    FComplete = -reshape(FComplete, [3, length(FComplete)/3])';
```

Comment:
It seems that you want to get only the y-components of the residual force. In this way, I would not go through this hassle, I would rather do the followind Fy = sum(FComplete(2:noSpatialDimensions:end,1)); which should already give you what you want